src/Microsoft.ComponentDetection.Detectors/go/Parsers/GoCLIParser.cs

```diff
@@ -74,19 +74,37 @@ public async Task<bool> ParseAsync(
     private void RecordBuildDependencies(string goListOutput, ISingleFileComponentRecorder singleFileComponentRecorder, string projectRootDirectoryFullName)
```

Comment:
If we didn't have explicit tests of this class before I think now might be a good time to add a couple.

Reply:
Good idea. I'll add some now.